server/spec/tests/repos/SettingsRepo.spec.ts

```diff
@@ -0,0 +1,47 @@
+import * as settingsRepo from '@src/repos/SettingsRepo'
+import mock from 'mock-fs'
```

Comment:
nit: Can we import this `as` `mockFs`? I'm worried that if we ever use an actual mocking library (seems likely if we keep writing tests), this `mock` will have a very misleading name, and `mockFs` makes what's happening clearer anyways